projects/canopy/src/lib/forms/select/docs/guide.stories.mdx

```diff
@@ -0,0 +1,104 @@
+import { Meta, Description } from '@storybook/addon-docs';
+import feedback from '../../../docs/common/feedback.md';
+
+<Meta title="Components/Form/Select/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Select
+
+<p class="standfirst">A drop down list that enables the user to choose between several options.</p>
+
+![example](docs/select/example.png)
+
+## Usage
+
+Use this component when users need to make a single selection from a list of options within a limited space. 
+
+Import the component in your application:
+
+```js
+@NgModule({
+  ...,
+  imports: [ ..., LgSelectModule ],
+})
+```
+
+and in your HTML:
+
+```html
+<lg-select-field [block]="block">
+  {{ label }}
+  <lg-hint *ngIf="hint">{{ hint }}</lg-hint>
+  <select lgSelect formControlName="color">
+    <option *ngFor="let option of options" [value]="option">{{ option }}</option>
+  </select>
+</lg-select-field>
+```
+
+## How it works
+
+The select component controls custom select box styling and linking the label and field.
+
+A default value can be pre-selected from the list, otherwise display an empty value of 'Select'.
+
+The entries in the list should follow a predictable format, such as alphabetical, chronological, or most frequently used.
+
+Like all form inputs, the width of select should match the maximum length of expected content. 
+
+When the user interacts with the select input, the select is opened and a list of options is revealed. The user can select an option by clicking on it with the mouse or using the keyboard, or tapping. The user can also scroll through the options using the up arrow or down arrow keys, or swiping.
+
+While the select box is styled by Canopy, the select menu styling is handled by the browser being used or according to the native OS styling if on mobile or tablet.
+
+## Dos and Don'ts
+
+### Do
+
+![do](docs/select/do.png)
+
+1. **Do** use a select component when the preset options is a long list (more than five) and space is limited
+2. **Do** list options in a predictable format, such as alphabetical, chronological, or most frequently used
+
+### Don't
+
+![dont](docs/select/dont.png)
+
+1. **Don't** use a select component when typing may be faster, such as date of birth or postcode; use Text input instead
+2. **Don't** use a select component as a way of presenting a single selection: use a Checkbox instead
+3. **Don't** use a select component when there are a large number of options; use autocomplete instead
+4. **Don't** use capitalised text, it's better to keep entries in sentence case for easy reading
+
+## Additional development details
+
```

Comment:
Could we add this line here please? **We recommend using <a href="https://angular.io/guide/reactive-forms" target="_blank">Angular's reactive forms</a>.**